<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScalarMappingChooser</class>
 <widget class="QDockWidget" name="ScalarMappingChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>258</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>scalar mapping</string>
  </property>
  <widget class="QWidget" name="content">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QLabel" name="relatedRenderView">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>(no render view selected)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="scalarsGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>scalars for color mapping</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="scalarsComboBox"/>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>min</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>max</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="minValueSpinBox">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="maxValueSpinBox">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>0.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gradientGroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>gradient</string>
      </property>
      <layout class="QVBoxLayout" name="gradientGroupLayout">
       <item>
        <widget class="QComboBox" name="gradientComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rearrangeDataObjectsBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>rearrange data objects...</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>gradientComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ScalarMappingChooser</receiver>
   <slot>gradientSelectionChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalarsComboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ScalarMappingChooser</receiver>
   <slot>scalarsSelectionChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minValueSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ScalarMappingChooser</receiver>
   <slot>minValueChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxValueSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ScalarMappingChooser</receiver>
   <slot>maxValueChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scalarsSelectionChanged(QString)</slot>
  <slot>gradientSelectionChanged(int)</slot>
  <slot>minValueChanged(double)</slot>
  <slot>maxValueChanged(double)</slot>
 </slots>
</ui>
